### DISCUSSION TITLE
Drug Types of Measurement Questions

### DISCUSSION THREAD

**Discussion by kntpricing** [CMSgov/hospital-price-transparency#144] [Q&A] [answered]
For drug type of measurement, we have drugs that have units that do not directly coincide with the valid values. For example, some drugs have measurement types of microgram, half tablet, tablet, packet, syringe, tablets, mEq, etc. How do we know which valid values should be assigned to these? I assume several of these should be listed as "EA" or "UN", what is the difference between these two values?

What are some examples of international units?

Thank you!
Kim

**Comment by RidenourK** [comment, marked as answer]
@kntpricing. Thank you for your questions. 

As indicated in the CY 2024 OPPS/ASC Final Rule, the current valid values reflect industry standards, specifically, we adopted both the [NDC standards](https://www.fda.gov/drugs/drug-approvals-and-databases/national-drug-code-directory) (which include UN (unit), ML (milliliter), GR (gram), F2 (International Unit), ME (milligram)) and the [NCPDP standards](https://www.ncpdp.org/NCPDP/media/pdf/BUS_fact_sheet.pdf) (which include ‘‘EA’’ (each), ‘‘ML’’ (milliliter), and ‘‘GM’’ (gram)). Please refer to the discussion at [88 FR 82103](https://www.federalregister.gov/d/2023-24293/p-5010).

Please refer to the [examples](https://github.com/CMSgov/hospital-price-transparency/tree/master/examples) of how you could encode drug unit and drug type of measurement, including micrograms.
